cli/azd/internal/appdetect/appdetect.go

```diff
@@ -154,10 +154,22 @@ func (a AzureDepServiceBus) ResourceDisplay() string {
 
 type AzureDepEventHubs struct {
 	EventHubsNamePropertyMap map[string]string
+	FromDependency           FromDependency
```

Comment:
nit: Could you please give it a better name? For example: `MavenDependency`.

Reply:
Updated.